packages/react/src/components/table/table-row.tsx

```diff
@@ -107,31 +107,6 @@ export const StyledTableRow = styled.tr<StyledTableRowProps>`
         }
     `}
 
-    ${({ $error, theme }) => $error && css`
```

Comment:
Buh bye, you won't be missed.

Ca fix le bug qu'on ne peut pas clicker dans le row quand il y a un erreur.